seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseSinkConfig.java

```diff
@@ -35,7 +33,7 @@
 import static org.apache.seatunnel.api.sink.DataSaveMode.DROP_DATA;
 import static org.apache.seatunnel.api.sink.DataSaveMode.ERROR_WHEN_DATA_EXISTS;
 
-public class BaseSinkConfig {
+public class BaseSinkConfig extends FileBaseOptions {
```

Comment:
Suggest we have an unified class name style. 
How about rename to 
- `FileBaseSinkOptions` 
- `FileBaseSourceOptions`
- `FileBaseOptions`